pkg/crc/api/api.go

```diff
@@ -69,12 +69,34 @@ func (api CrcApiServer) handleConnection(conn net.Conn) {
 		logging.Error("Error decoding request: ", err.Error())
 		return
 	}
-
-	if handler, ok := api.handlers[req.Command]; ok {
-		result := handler(req.Args)
-		writeStringToSocket(conn, result)
+	if req.Command == "start" || req.Command == "stop" || req.Command == "delete" {
+		// queue new request to channel
```

Comment:
The commit log says
" Issue #949 Make cluster operation calls serve in order

This patch makes start, stop and delete calls serve in order
and one at a time"

Why are these special, and why do we treat all other requests without the channel? Which problem are we trying to solve? This needs to be in the commit log